content/release_notes_chefdk.md

```diff
@@ -11,6 +11,164 @@ This page documents the ChefDK major changes for each release. For
 a detailed list of changes, see the [ChefDK Changelog on
 GitHub](https://github.com/chef/chef-dk/blob/master/CHANGELOG.md)
 
+## What's New In ChefDK 4.13
+
+Note: ChefDK will be enter end-of-life product phase Dec 31st 2020. The 4.13 release of ChefDK is the last planned release and all users should upgrade to Chef Workstation, which offers significant improvements to all included tools.
+
+### Package Updates
+
+ChefDK packages are no longer produced for Debian 8 and RHEL/CentOS 6 as these platforms are now both EOL.
+
+### Updated Components
+
+#### Test Kitchen
+
+Test Kitchen was updated from 2.7.2 to 2.8.0. This release improves how we execute commands on Windows hosts in order to avoid failures executing commands that are too long for the windows command line. Thanks for this fix [@ramereth](https://github.com/ramereth)!
```

Comment:
```suggestion
Test Kitchen was updated from 2.7.2 to 2.8.0. This release improves how we execute commands on Windows hosts in order to avoid failures from executing commands that are too long for the windows command line. Thanks for this fix [@ramereth](https://github.com/ramereth)!
```